Gemfile

```diff
@@ -1,2 +1,5 @@
 source "https://www.rubygems.org"
 gemspec
+
+gem 'daru', :path => "/home/sameer/gitrepos/daru/"
```

Comment:
I know that, but you should redirect that URL to a git branch in your repository. That way, you don't pollute your master branch (you can always rebase before merging) and it would allow Travis to test this pull request.

See [http://bundler.io/git.html](http://bundler.io/git.html).